src/client/components/pages/NonClassMeetings/NonClassMeetingsTable.tsx

```diff
@@ -0,0 +1,244 @@
+          area,
+          title,
+          expectedSize,
+          notes,
+          contactEmail,
+          contactName,
+          contactPhone,
+          spring,
+          fall,
+        }, index) => (
+          <TableRow isStriped={index % 2 !== 0} key={id}>
+            <TableCell backgroundColor={getAreaColor(area)} key={`area-${id}`}>{area}</TableCell>
+            <TableCell key={`title-${id}`}>{title}</TableCell>
+            <TableCell key={`expectedSize-${id}`}>{expectedSize}</TableCell>
+            <TableCell key={`notes-${id}`}>{notes}</TableCell>
+            <TableCell key={`contactInfo-${id}`}>
+              {
+                /*
+                  * This is being used instead of just outputting the fields
+                  * directly as a way to filter out any null values. This
+                  * ensures that if a phone number, name or email is missing
+                  * we don't end up with blank lines where the "<br />"s still
+                  * are, but with no data before them.
+                  *
+                  * The same result could be accomplished with lots of ternary
+                  * logic, but this felt like a cleaner (and more scalable) way
+                  * to solve the problem.
+                  */
+                [
+                  contactName,
+                  contactEmail,
+                  contactPhone,
+                ]
+                  .filter((info) => !!info)
+                  .map((info) => (<p key={info}>{info}</p>))
+              }
+            </TableCell>
+            <TableCell key={`fallMeetings-${id}`}>
+              <CellLayout>
+                <TableCellList>
+                  {fall.meetings.map(({
+                    id: meetingId,
+                    day,
+                    startTime,
+                    endTime,
+                    room,
+                  }) => (
+                    <TableCellListItem key={meetingId}>
+                      <MeetingGrid>
+                        <MeetingGridSection area="time">
+                          <div>{dayEnumToString(day)}</div>
+                          <div>{`${startTime}-${endTime}`}</div>
+                        </MeetingGridSection>
+                        {room && (
+                          <>
+                            <MeetingGridSection area="room">
+                              {room.name}
+                            </MeetingGridSection>
+                            <MeetingGridSection area="campus">
+                              <CampusIcon>{room.campus}</CampusIcon>
+                            </MeetingGridSection>
+                          </>
+                        )}
+                      </MeetingGrid>
+                    </TableCellListItem>
+                  ))}
+                </TableCellList>
+                <BorderlessButton
+                  id={`${id}-${spring.term}-edit-meetings-button`}
+                  key={`${id}-${spring.term}-edit-meetings-button`}
```

Comment:
I have the same question as above for this line regarding the `key`s.

Reply:
Fixed in